src/plugins/borrow-plugins/plugins/aave/AaveNetwork.js

```diff
@@ -15,7 +15,8 @@ export type AaveNetworkBlueprint = {
   contractAddresses: {
     lendingPool: string,
     protocolDataProvider: string
-  }
+  },
+  enabledTokens: { [sybmbol: string]: boolean }
```

Comment:
Please add a comment that these are currencyCodes, not tokenId's. Or, change `symbol` to `currencyCode` Ideally we would be using tokenId's for everything, but as long as this is an internal implementation detail, it's ok.

Reply:
Done